packages/ckeditor5-engine/src/view/renderer.ts

```diff
@@ -826,9 +882,11 @@ export default class Renderer extends Observable {
 		// Render selection.
 		if ( this.selection.isFake ) {
```

Comment:
The chain of these conditions is hard for me to understand.

I also can't understand the difference between what we had before and what's there now.

So far it seems that we were calling `updateDomSelection()` when `isComposing` was `true`. Right now we don't do that (or more precisely: do this only when there was fakeSelContainer). This sounds like a big change compared to what we had before. Could you explain why you made it (in this PR)?

Reply:
PS. And most of that needs to end up in the comments :D

Reply:
https://github.com/ckeditor/ckeditor5/blob/2b9e30f34e5b87c0dd27613a3e6477129798e111/packages/ckeditor5-engine/src/view/renderer.ts#L897-L913